### PR TITLE
Fixed Twitter demo for App Builder

### PR DESCRIPTION
**Pull request by ilooner**

### CODE REVIEW

File: demos/pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
   </modules>
 
   <properties>
-    <datatorrent.version>3.0.0</datatorrent.version>
+    <datatorrent.version>3.0.1-SNAPSHOT</datatorrent.version>
```
Comment (tweise) [inline demos/pom.xml]: Why this change?

Comment (ilooner) [inline demos/pom.xml]: Otherwise the demos will try to use 3.0.0 version of malhar-contrib. If the 3.0.0 version of contrib is used then the old version of the TwitterSampler operator will be included in the app package and the properties will not be displayed in app builder.

Comment (tweise) [inline demos/pom.xml]: That's an existing mistake in the pom.xml. datatorrent.version refers to
the engine version. Use project.version to refer to the Malhar version.

Comment (ilooner) [inline demos/pom.xml]: I've updated the dependencies to use the project.version and have kept the datatorrent.version at 3.0.0 . I noticed that the other demos in Malhar had this issue as well. I can open a separate pull request to update those as well if necessary.

**Comment by tweise** [comment]
cherry picked to 3.1.0

**Comment by 243826** [comment]
This is not 3.0.1 compatible change. Please don't cherry pick. Raise pull requests, test those, and pull those in.

So revert both thie commits. Revert cherry pick and pull this in 3.1.0.

**Comment by tweise** [comment]
Let's talk about this offline.

--
sent from mobile
